test/credit_card_sanitizer_test.rb

```diff
@@ -66,6 +66,11 @@ class CreditCardSanitizerTest < MiniTest::Test
         end
       end
 
+      it "doesn't fail if text is not utf-8 encoded" do
+        ascii_text = '41111111111111112'.force_encoding(Encoding::ASCII)
```

Comment:
this test also passed without the new code, the problem we are trying to solve is that a with invalid utf8 that is encoded in another encoding was not stripped correctly

```
      it "doesn't fail if the text contains invalid utf-8 characters but is not in utf-8" do
        if ''.respond_to?(:encoding)
          invalid_characters = "你好 4111 1111 1111 1111 \255there".force_encoding(Encoding::BINARY)
          assert_equal "你好 4111 11▇▇ ▇▇▇▇ 1111 \ufffdthere", @sanitizer.sanitize!(invalid_characters)
        end
      end
```